### PR TITLE
[MB-1198] Adding configurations for message cache

### PR DESCRIPTION
**Pull request by ramith**
Jira: https://wso2.org/jira/browse/MB-1198

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/MessageCacheConstants.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+package org.wso2.andes.store.cache;
```
Comment (abeykoon) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/MessageCacheConstants.java]: License header mssing?

File: modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/GuavaBasedMessageCacheImpl.java [hunk 2]
```diff
@@ -0,0 +1,174 @@
+package org.wso2.andes.store.cache;
```
Comment (abeykoon) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/GuavaBasedMessageCacheImpl.java]: Let's add license header here

File: modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/MessageCacheConstants.java (hunk 1 above)
Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/MessageCacheConstants.java]: hell yes!!! this is me being extremely bad. apologies.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/GuavaBasedMessageCacheImpl.java (hunk 2 above)
Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/GuavaBasedMessageCacheImpl.java]: done

File: modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/MessageCacheConstants.java (hunk 1 above)
Comment (ramith) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/store/cache/MessageCacheConstants.java]: done

**Comment by ramith** [comment]
shall we merge?

**Pull request merged**